iib/web/models.py

```diff
@@ -810,8 +810,13 @@ def from_json(cls, kwargs):
             if not isinstance(request_kwargs[param], str):
                 raise ValidationError(f'"{param}" must be a string')
 
-        # Always remove cnr_token from request_kwargs since it's not stored in the database
+        if 'force_backport' in request_kwargs:
+            if not isinstance(request_kwargs['force_backport'], bool):
```

Comment:
In the future, I prefer this syntax:
```python
if not isinstance(request_kwargs.get('force_backport', False), bool):
    raise ValidationError('"force_backport" must be a boolean')
```

Reply:
I do too, not sure why I didn't do that. Here's a PR to change it over: https://github.com/release-engineering/iib/pull/115